frontend/src/components/Forms/InterestsForm/index.module.css

```diff
@@ -1,4 +1,6 @@
 .selectedInterestsWrapper {
   display: grid;
-  row-gap: 24px;
+  grid-template-columns: repeat(2, 158px);
+  grid-gap: 20px;
+  margin-bottom: 32px;
```

Comment:
We can move this out to `FormLayout` by creating a `main` class and wrapping the `children`. Make sure to remove it everywhere else too.